### PR TITLE
Test #99 popup dialog

### LINKED ISSUE #99: (SP: 1) Write unit test for "PopupDialog" component
### Component unit test
Unit test for "**PopupDialog**" component
Scenaries descriptions:
- [x] Content text should be shown
- [x] Imitate click on the close button and popup should be closed
- [x] Imitate click with delay and popup should be closed

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/components/popup-dialog/PopupDialog.jsx)

Current coverage:

Image

### PR DESCRIPTION
**Pull request by AndriiNasadiuk**
close #99

### CODE REVIEW

**Review by marianamartsy — COMMENTED** [review]
will you change this code like we talked on meeting?

**Comment by AndriiNasadiuk** [comment]
> will you change this code like we talked on meeting?

Maybe later, not now)

**Review by ArtemHolikov — APPROVED** [review]
Seems correctly. but don`t forget to delete Loader from this commit before merge.

**Comment by vlad-khrychov** [comment]
In task says about "closing" of modal window, but I don`t see test about it (`not.toBeInTheDocument()`)

**Review by Mygal-Alex — APPROVED** [review]
bug#131Sign-in/Sign-up

**Review by marianamartsy — APPROVED** [review]
👍

**Review by LeonidShamarin — APPROVED** [review]

File: tests/unit/components/loader/Loader.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/components/loader/Loader.spec.jsx:1]: Delete this empty file from pull request

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: tests/unit/components/loader/Loader.spec.jsx
Comment (AndriiNasadiuk) [reply tests/unit/components/loader/Loader.spec.jsx:1]: Done

**Pull request merged**